_drafts/let-it-throw.md

```diff
@@ -137,32 +137,32 @@ func doTheActualCall() throws {
 
 ## Don't let them in, don't let them see
 
-For that reason, but also to avoid your internal errors to bleed across your library boundaries and to limite the number of error types to handle by your user, I suggest that you keep your error types scoped to each level of abstraction.
+For that reason, but also to prevent your internal errors from bleeding across your library boundaries and to limit the number of error types that must be handled by your users, I suggest that you keep your error types scoped to each level of abstraction.
 
-In the example above, instead of making `funcC` directly propagade both `MyLibAErrors` and `MyLibBErrors`, you should instead throw `MyLibCErrors`. I suggest that for two reasons, both related to abstraction:
+In the example above, instead of making `funcC` directly propagate both `MyLibAErrors` and `MyLibBErrors`, you should instead throw `MyLibCErrors`. I suggest this for two reasons, both related to abstraction:
 
-* The first one is because your user shouldn't have to know which internal library you're using. If some day in the future you decide to switch your implementation to use `SomeOtherPopularLibA` instead of `MyLibA`, which will obviously not throw exactly the same errors, the caller of your own `MyLibC` framework should care, that's what abstraction is all about.
-* The second one is because the caller shouldn't have to handle all the errors. Surely you can catch some of those error and consider them internal: not all errors thrown by `MyLibA` do make sense to be exposed publically to your user, for example a `FrameworkConfigurationError` error mentionning that you misused the `MyLibA` framework and forgot to call its `setup()` method or whatever should not make its way to the user, as the user can't do much about it, that kind of error is your fault, not his.
+1. Your users shouldn't have to know which internal library you're using. If some day in the future you decide to switch your implementation to use `SomeOtherPopularLibA` instead of `MyLibA`, which will obviously not throw exactly the same errors, the caller of your own `MyLibC` framework shouldn't need to know or care. That's what abstraction is all about.
+1. The caller shouldn't have to handle all the errors. Surely you can catch some of those errors and consider them internal: not all errors thrown by `MyLibA` make sense to be exposed publicly to your users, for example a `FrameworkConfigurationError` error mentioning that you misused the `MyLibA` framework and forgot to call its `setup()` method or whatever should not make its way to the user, as the user can't do much about it. That kind of error is your fault, not his/hers.
```

Comment:
:+1: even better. I hate how English doesn't have a gender-neutral pronoun. "y'all" is the best we have.